README.md

```diff
@@ -347,8 +347,9 @@ execute(function(finished) {
 });
 
 if (fireNow) {
-	// trigger the event when smoke testing or run locally for testing
-	events.emit('my-event', 'MSFT');
+    // trigger the event when smoke testing or run locally for testing
+    events.emit('my-event', 'MSFT');
+    events.finish()
```

Comment:
This is only for smoke testing or running locally as indicated by the comment two lines above. Need the same thing in the actual example:

```
events.on('my-event', function(symbol) {
    request.get('http://sample.testable.io/stocks/' + symbol);
    events.finished();
    finished();
  });
```